packages/serverpod/lib/src/server/session.dart

```diff
@@ -612,6 +613,42 @@ class MessageCentralAccess {
   /// emit an error.
   Stream<T> createStream<T>(String channelName) =>
       _session.server.messageCentral.createStream<T>(_session, channelName);
+
+  /// Broadcasts revoked authentication events to the Serverpod framework.
+  /// This message ensures authenticated connections to the user are closed.
+  ///
+  /// The [userId] should be the [AuthenticationInfo.userId] for the concerned
+  /// user.
+  ///
+  /// The [message] must be of type [RevokedAuthenticationUser],
+  /// [RevokedAuthenticationAuthId], or [RevokedAuthenticationScope].
+  ///
+  /// [RevokedAuthenticationUser] is used to communicate that all the user's
+  /// authentication is revoked.
+  ///
+  /// [RevokedAuthenticationAuthId] is used to communicate that a specific
+  /// authentication id has been revoked for a user.
+  ///
+  /// [RevokedAuthenticationScope] is used to communicate that a specific
+  /// scope or scopes have been revoked for the user.
+  Future<bool> revokedAuthentication(
```

Comment:
Suggestion: Use the name `authenticationRevoked` to make it sound more like an event that happened rather than some you are doing. I get that the difference here is super small.

Reply:
Similar to my comment here: https://github.com/serverpod/serverpod/pull/2679#discussion_r1732927313

> Suggestion: since an action is taken below (postMessage), perhaps this should be named revokeAuthentication (imperative command), instead of past tense?

Reply:
Good suggestion! will fix!